roadmap.md

```diff
@@ -18,27 +20,25 @@ At Microsoft, .NET has always been an important component of other teams' produc
 
 |Milestone|Release Date|
 |---------|------------|
-|1.0      |   June 2016|
+|1.0      |   Spring 2016|
```

Comment:
I think we can just delete 1.0 at this point. The roadmap isn't intended to be historical in nature.